src/main/scala/elevate/rise/rules/lowering.scala

```diff
@@ -245,7 +248,9 @@ object lowering {
 
   // todo gotta use a normalform for introducing copies! e.g., if we have two reduce primitives
   def lowerToC(implicit ev: Traversable[Rise]): Strategy[Rise] =
-    addRequiredCopies `;` `try`(bottomUp(copyAfterReduceInit)) `;` specializeSeq
+    debug[Rise]("before lowering") `;`
+    addRequiredCopies `;` specializeSeq `;` debug("after lowering")
```

Comment:
are we keeping these debug prints in the PR?

Reply:
I will removed them